rust/sink/README.md

```diff
@@ -2,4 +2,20 @@
 
 Is a specialized library to handle data from the nasl-interpreter to the storage / distribution implementation.
 
-The main purpose is to make handling of the data easy for the interpreter while the storage / distribution implementation can be unaware of the requirements based on the data.
+To be able to introduce new distribution implementations the [Sink](src/lib.rs#L80) must be i implement.
```

Comment:
Not sure about the links to specific code lines. Is there a way to link to functions?

Reply:
unfortunately not; the next I found is using a browser functionality to search for text but that has the same disadvantage but is more complex.